### PR TITLE
Add missing controllers and broadcasters to ros2_controllers dependencies

### PR DESCRIPTION
**Pull request by saikishor**
This PR adds missing dependencies of new controllers and broadcasters in ros2_controllers package

### CODE REVIEW

File: ros2_controllers/package.xml [hunk 1]
```diff
@@ -25,13 +25,17 @@
...
   <exec_depend>imu_sensor_broadcaster</exec_depend>
   <exec_depend>joint_state_broadcaster</exec_depend>
   <exec_depend>joint_trajectory_controller</exec_depend>
   <exec_depend>mecanum_drive_controller</exec_depend>
+  <exec_depend>parallel_gripper_controller</exec_depend>
   <exec_depend>pid_controller</exec_depend>
+  <exec_depend>pose_broadcaster</exec_depend>
   <exec_depend>position_controllers</exec_depend>
   <exec_depend>range_sensor_broadcaster</exec_depend>
+  <exec_depend>rqt_joint_trajectory_controller</exec_depend>
```
Comment (saikishor) [inline ros2_controllers/package.xml:38]:
```suggestion
```

**Review by christophfroehlich — APPROVED** [review]
Thx!

**Comment by christophfroehlich** [comment]
Humble needs a different patch, we don't have the parallel gripper controller there.

**Comment by saikishor** [comment]
> Humble needs a different patch, we don't have the parallel gripper controller there.

Sure. I can create a backport manually.

**Pull request merged**